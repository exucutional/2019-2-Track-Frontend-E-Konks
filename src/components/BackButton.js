/* eslint-disable react/destructuring-assignment */
import React from 'react';
import styled from '@emotion/styled';

const Button = styled.img`
	display: flex;
	height: 8vh;
	margin-left: 10px;
	filter: invert(1);
	&:hover {
		filter: invert(0.5);
	}
`;

function BackButton(props) {
	return (
		<Button
			src="https://image.flaticon.com/icons/svg/109/109618.svg"
		/>
	);
}


export default BackButton;
