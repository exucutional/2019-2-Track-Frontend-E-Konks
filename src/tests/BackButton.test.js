import React from 'react';
import renderer from 'react-test-renderer';
import BackButton from '../components/BackButton'

it('renders correctly', () => {
	const tree = renderer
		.create(<BackButton/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
