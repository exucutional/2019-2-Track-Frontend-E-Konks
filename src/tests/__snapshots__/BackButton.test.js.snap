// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-z5psf0"
  src="https://image.flaticon.com/icons/svg/109/109618.svg"
/>
`;
